server/src/app.js

```diff
@@ -6930,7 +6930,7 @@ export async function pp_init(serverconfig) {
 			/*
 		for each raw dataset
 		*/
-
+			if (d.skip) continue
```

Comment:
do you put `"skip":1` in serverconfig? i would suggest just moving a dataset entry between `"tmp":[]` and `"datasets":[]` to do it, that's easier than writing/deleting a property, and also can get rid of this line in app.js (already complicated)